client/state/terms/test/actions.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * External dependencies
+ */
+import sinon from 'sinon';
+import { expect } from 'chai';
+import nock from 'nock';
+
+/**
+ * Internal dependencies
+ */
+import {
+	TERMS_RECEIVE,
+	TERMS_REQUEST,
+	TERMS_REQUEST_SUCCESS,
+	TERMS_REQUEST_FAILURE
+} from 'state/action-types';
+import {
+	receiveTerms,
+	requestSiteTerms
+} from '../actions';
+
+/**
+ * Module Variables
+ */
+const testTerms = [
+	{ ID: 777, name: 'Chicken', slug: 'chicken', description: 'cornel sanders', post_count: 1, number: 0 },
+	{ ID: 778, name: 'Ribs', slug: 'ribs', description: 'i want my baby back * 3 ribs', post_count: 100, number: 0 },
+];
+const siteId = 771234;
```

Comment:
Site IDs in this file, while seemingly random, are associated with private user sites (`12345`, `771234`). Suggest instead one of our preferred example site IDs:

- `2916284`
- `77203074`
- `77203199`